ui/raidboss/data/05-shb/raid/e2n.js

```diff
@@ -24,42 +24,30 @@
   triggers: [
     {
       id: 'E2N Shadowflame Tank',
-      regex: Regexes.startsUsing({ id: '3E4D', source: 'Voidwalker', capture: false }),
-      regexDe: Regexes.startsUsing({ id: '3E4D', source: 'Nichtswandler', capture: false }),
-      regexFr: Regexes.startsUsing({ id: '3E4D', source: 'Marcheuse Du Néant', capture: false }),
-      regexJa: Regexes.startsUsing({ id: '3E4D', source: 'ヴォイドウォーカー', capture: false }),
-      regexCn: Regexes.startsUsing({ id: '3E4D', source: '虚无行者', capture: false }),
-      regexKo: Regexes.startsUsing({ id: '3E4D', source: '보이드워커', capture: false }),
+      regex: Regexes.startsUsing({ id: '3E4D', source: 'Voidwalker' }),
+      regexDe: Regexes.startsUsing({ id: '3E4D', source: 'Nichtswandler' }),
+      regexFr: Regexes.startsUsing({ id: '3E4D', source: 'Marcheuse Du Néant' }),
+      regexJa: Regexes.startsUsing({ id: '3E4D', source: 'ヴォイドウォーカー' }),
+      regexCn: Regexes.startsUsing({ id: '3E4D', source: '虚无行者' }),
+      regexKo: Regexes.startsUsing({ id: '3E4D', source: '보이드워커' }),
       condition: function(data) {
         return data.role == 'tank';
       },
-      alertText: {
-        en: 'Tank Buster on YOU',
-        de: 'Tankbuster auf DIR',
-        fr: 'Tankbuster sur VOUS',
-        cn: '死刑点名',
-        ko: '나에게 탱버',
-      },
+      response: Responses.tankBuster(),
     },
     {
       id: 'E2N Shadowflame Healer',
-      regex: Regexes.startsUsing({ id: '3E4D', source: 'Voidwalker', capture: false }),
-      regexDe: Regexes.startsUsing({ id: '3E4D', source: 'Nichtswandler', capture: false }),
-      regexFr: Regexes.startsUsing({ id: '3E4D', source: 'Marcheuse Du Néant', capture: false }),
-      regexJa: Regexes.startsUsing({ id: '3E4D', source: 'ヴォイドウォーカー', capture: false }),
-      regexCn: Regexes.startsUsing({ id: '3E4D', source: '虚无行者', capture: false }),
-      regexKo: Regexes.startsUsing({ id: '3E4D', source: '보이드워커', capture: false }),
+      regex: Regexes.startsUsing({ id: '3E4D', source: 'Voidwalker' }),
+      regexDe: Regexes.startsUsing({ id: '3E4D', source: 'Nichtswandler' }),
+      regexFr: Regexes.startsUsing({ id: '3E4D', source: 'Marcheuse Du Néant' }),
+      regexJa: Regexes.startsUsing({ id: '3E4D', source: 'ヴォイドウォーカー' }),
+      regexCn: Regexes.startsUsing({ id: '3E4D', source: '虚无行者' }),
+      regexKo: Regexes.startsUsing({ id: '3E4D', source: '보이드워커' }),
       suppressSeconds: 1,
       condition: function(data) {
         return data.role == 'healer';
       },
-      infoText: {
-        en: 'tank busters',
-        de: 'tank buster',
-        fr: 'Tank busters',
-        cn: '死刑',
-        ko: '탱버',
-      },
+      response: Responses.tankBuster(),
```

Comment:
This is "multiple tankbusters".  tankBuster on its own here will only call out tankbuster on one person, which I think is confusing.